101-bundle-json.md

```diff
@@ -196,6 +197,20 @@ The following OPTIONAL fields MAY be attached to an invocation image:
   - `os`: The operating system of the image
 - `mediaType`: The media type of the image
 
+### Resolving Image References
+
+With invocation images and images that use the OCI image format, the referenced image is of the format `REGISTRY/IMAGE:TAG@DIGEST`.
+
+`REGISTRY` is optional. `IMAGE` is required. `TAG` is optional. `DIGEST` is optional, but strongly recommended.
+
+- If `IMAGE` is not specified, the runtime MUST fail with an error.
+- When `DIGEST` is specified, the runtime MUST use the version specified by that digest and MUST fail if an image of that digest cannot be found.
+- When no `DIGEST` is specified, but a `TAG` is specified, the runtime MUST use an image with that tag or fail.
+- When neither `DIGEST` nor `TAG` is defined, the behavior is undefined.
+- If `REGISTRY` is missing, a runtime SHOULD assume `dockerhub.io` is the registry. This matches the behavior of existing clients.
```

Comment:
Is `dockerhub.io` the actual registry domain for Docker Hub?

Reply:
Good catch. No, the proper domain is `docker.io` (or if you're an OG, `index.docker.io`).

Reply:
I believe the default is docker.io 
https://github.com/moby/moby/blob/8e610b2b55bfd1bfa9436ab110d311f5e8a74dcb/registry/config.go#L36
```
docker pull docker.io/library/hello-world:latest
latest: Pulling from library/hello-world
Digest: sha256:2557e3c07ed1e38f26e389462d03ed943586f744621577a99efb77324b0fe535
Status: Image is up to date for hello-world:latest
```

Reply:
Snarky :bacon: :turkey: